zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Page.java

```diff
@@ -291,9 +291,16 @@ public void deletePage(HttpServletRequest request, HttpServletResponse response)
         }
 
         try {
+            String uriToDelete = uri;
+            // Remove trailing slashes
+            while (uriToDelete.endsWith("/")) {
+                uriToDelete = uriToDelete.substring(0, uriToDelete.length() - 1);
+            }
```

Comment:
This could potentially be simplified as:
```suggestion
           String uriToDelete = uri.replaceAll("/+$","");
```